__tests__/posts.ts

```diff
@@ -6816,7 +6837,7 @@ describe('Source post moderation approve/reject', () => {
     await saveSquadFixtures();
     await con.getRepository(SourcePostModeration).save([
       {
-        id: pendingId,
+        id: pendingId as string,
```

Comment:
Why are `as string` cases needed?

Reply:
it was throwing me an error previously :triggered:, but when I removed it now, it doesn't.
Something about `string` not matching 
~~~
`${string}-${string}-${string}-${string}-${string}`
~~~